install/2.7Scripts/2.7.1Script/generic-mirror.sh

```diff
@@ -19,7 +19,9 @@ cat <<EOF
 Usage: $(basename "${BASH_SOURCE[0]}") <args>
 
 Prerequisites Required:
-    Minimum Skopeo version should be 1.14
+    - Docker or Podman should be installed
```

Comment:
@anvesh-thangallapalli we need to point to KC document mirroring host setup steps here so all prerequisites are there https://www.ibm.com/docs/en/sfhs/2.7.x?topic=installation-mirroring-your-images-enterprise-registry#tasktask_htj_h1w_stb__prereq__1

Reply:
@ganshug done